build/common.sh

```diff
@@ -48,9 +48,7 @@ readonly KUBE_GCS_DELETE_EXISTING="${KUBE_GCS_DELETE_EXISTING:-n}"
 
 # Constants
 readonly KUBE_BUILD_IMAGE_REPO=kube-build
-readonly KUBE_BUILD_GOLANG_VERSION=1.4.2
-readonly KUBE_BUILD_IMAGE_CROSS_TAG="cross-${KUBE_BUILD_GOLANG_VERSION}-2"
-readonly KUBE_BUILD_IMAGE_CROSS="${KUBE_BUILD_IMAGE_REPO}:${KUBE_BUILD_IMAGE_CROSS_TAG}"
+readonly KUBE_BUILD_IMAGE_CROSS_TAG=v1
```

Comment:
We should probably still tag the cross-build image with the golang version as a first order bit so it's obvious, shouldn't we? (That seems like the largest piece versioned component of the image.) e.g. `v1.4.2-1`